tasks/main.yml

```diff
@@ -39,6 +39,9 @@
     external: "{{ item.external | default(omit) }}"
     project: "{{ item.project | default(omit) }}"
     state: "{{ item.state | default(omit) }}"
+    mtu: "{{ item.mtu | default(omit) }}"
```

Comment:
I think if you try to run this on Ansible 2.8 it will fail. That was behaviour previously at least

Reply:
It claims to have been added in 2.9, according to https://docs.ansible.com/ansible/latest/modules/os_network_module.html - however if I try it on a production system I get:

`BadRequestException: 400: Client Error for url: https://ostk.sib.swiss:9696/v2.0/networks.json, Invalid input for operation: Requested MTU is too big, maximum is 1500.`

... although setting the MTU to 9000 on the CLI appears to work fine.